server/src/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -767,7 +767,9 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
                 // set up a new alert
                 AlertVO newAlert = new AlertVO();
                 newAlert.setType(alertType.getType());
-                newAlert.setSubject(subject);
+                //do not have a seperate column for content.
+                //appending the message to the subject for now.
+                newAlert.setSubject(subject+content);
```

Comment:
@rafaelweingartner 
I agree with you that it is a half measure, And so i commented "appending the message to the subject **for now**."  The idea was just to improve it little by little. We need not do every thing in this PR. We can work on it in a separate one.